modules/javafx.web/src/main/native/Source/JavaScriptCore/CMakeLists.txt

```diff
@@ -181,6 +181,32 @@ file(GLOB JavaScriptCore_SCRIPTS_SOURCES ${JavaScriptCore_SCRIPTS_SOURCES_PATHS}
 
 foreach (_file ${JavaScriptCore_SCRIPTS_SOURCES})
     get_filename_component(_script "${_file}" NAME)
+    #the copy_if_different  is failing on Mac OSX 11 platform if build using -j12
+    #Introducing sleep for 1 s and retry file copy 10 times, to prevent copy failing error
+    if (PORT STREQUAL "Java")
+    # introduce one second delay to prevent any lock
+    execute_process(COMMAND ${CMAKE_COMMAND} -E sleep 1)
+    set(copy_success FALSE)
+    set(copy_attempts 0)
+    while(NOT copy_success AND copy_attempts LESS 10)
+        execute_process(
+            COMMAND ${CMAKE_COMMAND} -E copy_if_different ${_file} ${JavaScriptCore_SCRIPTS_DIR}/${_script}
+            RESULT_VARIABLE copy_result
+            OUTPUT_VARIABLE copy_output
+        )
+        if(copy_result)
+            message(WARNING "Failed to copy ${_file} to ${JavaScriptCore_SCRIPTS_DIR}/${_script}: ${copy_output}")
```

Comment:
Shouldn't `${copy_output}` at the end of this message be `${copy_result}`?

If not, I think it would be worth to print the `${copy_result}` variable regardless, in case there are any issues with the copy process.

Reply:
do you mean as 
if(copy_result)
            message(WARNING "Failed to copy ${_file} to ${JavaScriptCore_SCRIPTS_DIR}/${_script}: ${copy_output}")
should be as

if(copy_result)
            message(WARNING "${copy_result")

Reply:
Actually, you can disregard this comment - I thought that `cmake -E copy_if_different ...` returns some meaningful error code (ex. one of errno variables) but I double-checked and it seems it returns 0 if succeeded and 1 if failed. Won't bring us any good to print it out, so it can stay as it is.